### DISCUSSION TITLE
How to get text-primary-content as variables?

### DISCUSSION THREAD

**Discussion by Ashvith10** [saadeghi/daisyui#1476] [Q&A] [answered]
I know that we can get the value of primary color using `hsl(var(--p))`, however, I haven't been able to find anything similar for modifier classes. I need this value so that I can set this value as the SVG fill color for the navbar logo.

**Comment by Ashvith10** [comment]
My bad, I should have read the docs clearly, although there wasn't any variable for `text-primary-content`, there was one for `primary-content`. I am assuming that the former is just an alias for better readability.

**Comment by saadeghi** [comment, marked as answer]
`primary` is `hsl(var(--p))` and `primary-content` is `hsl(var(--pc))`  
You can see all colors and their CSS variables here: https://daisyui.com/docs/colors/  
`text-` prefix is just a Tailwind CSS class. So when we use `text-primary` it will result `color` to `primary` in CSS or when we use `bg-primary` it sets `background-color`
